app/lib/features/pins/pages/pin_page.dart

```diff
@@ -115,14 +93,52 @@ class PinPage extends ConsumerWidget {
       );
     }
 
-    if (actions.isEmpty) {
-      return const SizedBox.shrink();
-    }
+    return Visibility(
+      visible: actions.isNotEmpty,
+      child: PopupMenuButton<String>(
+        key: PinPage.actionMenuKey,
+        icon: const Icon(Atlas.dots_vertical_thin),
+        itemBuilder: (ctx) => actions,
+      ),
+    );
+  }
```

Comment:
I understand this code was there before in a similar matter _but_ this actions may never be empty: If you can't edit or redact the item, you can always report it. The `if` above might not be 100% correct about this, but overall the menu is always existing...

```suggestion
    return PopupMenuButton<String>(
        key: PinPage.actionMenuKey,
        icon: const Icon(Atlas.dots_vertical_thin),
        itemBuilder: (ctx) => actions,
    );
  }
```